### PR TITLE
target-bigquery-repo-transfer

### PR DESCRIPTION
**Pull request by pnadolny13**
Closes https://github.com/meltano/hub/issues/1014

Did I do this right? I left transferwise as hidden and made a new one for the new maintainer repo.

### CODE REVIEW

File: _data/meltano/loaders/target-bigquery/jmriego.yml [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+  label: Validate Records
+- name: temp_schema
+  description:
+    Name of the schema where the temporary tables will be created. Will
+    default to the same schema as the target tables.
+  label: Temp Schema
+settings_group_validation:
+- - dataset_id
+  - project_id
+pip_url: pipelinewise-target-bigquery
```
[the remark this answers is not in the document]
Comment (edgarrmondragon) [reply _data/meltano/loaders/target-bigquery/jmriego.yml:136]: I'd keep the PyPI reference. The maintainer has been publishing there.

**Review by edgarrmondragon — APPROVED** [review]

**Comment by edgarrmondragon** [comment]
> Did I do this right? I left transferwise as hidden and made a new one for the new maintainer repo.

@pnadolny13 Yeah, we gotta keep the transferwise variant 👍.

Should we add `jmriego` as a new maintainer?

**Review by tayloramurphy — CHANGES_REQUESTED** [review]
@pnadolny13 can you update the maintenance_status of the transferwise one to Inactive (or deprecated if we have that)?

Also, updated the next_steps section and/or description to note that the repo was transferred would be helpful for users too.

**Comment by pnadolny13** [comment]
> Should we add jmriego as a new maintainer?

Yes but interesting that the tests didnt fail for this, they are supposed to check for all maintainers existing. 🤔

**Review by tayloramurphy — APPROVED** [review]

**Pull request merged**